packages/rum/README.md

```diff
@@ -253,6 +253,38 @@ init(configuration: {
 })
 ```
 
+### Browser and Session Replay sampling and configuration
```

Comment:
Could it be interesting to link related [init parameters](https://docs.datadoghq.com/real_user_monitoring/browser/#initialization-parameters) to this section?

Reply:
I'm not sure I understand what you're thinking, do you have more details?

Reply:
we have documentation for all initialization parameters, for sampleRate and replaySampleRate, it could be nice to add a "more details" link to this new paragraph.

Reply:
```suggestion
### Browser and Session Replay sampling and configuration

For more details about `sampleRate` and `replaySampleRate`, see [Initialization parameters](#initialization-parameters).
```

Reply:
@bcaudan I understand your ask but we're adding this new section immediately after Initialization Parameters—do you think this "more details" link is necessary?

Reply:
What I had in mind was not what you suggested above but having in the sample rate and replay sample rate doc a link to Browser and Session Replay sampling and configuration. ex:

```md
`sampleRate`
: Optional<br/>
**Type**: Number<br/>
**Default**: `100`<br/>
The percentage of sessions to track: `100` for all, `0` for none. Only tracked sessions send rum events.
[More details on sampling configuration](#browser-and-session-replay-sampling-and-configuration).
```
When browsing initialization parameters, it can be nice to see that there are extra details somewhere.

Reply:
Got it, so you want to add the link to the section in the table entries—in that case, I agree! @MirandaKapin can you add this to the two entries in the table: 

For more details about `sampleRate` and `replaySampleRate`, see [the sampling configuration](#browser-and-session-replay-sampling-and-configuration).

Reply:
added in 63c0fb4